Wrappers/Python/ccpi/viewer/utils/example_data.py

```diff
@@ -0,0 +1,63 @@
+import os
+import os.path
+import vtk
+import sys
+
+# this is the default location after pip install
```

Comment:
I suppose this comment should relate to the location of the data coming from `cil-data`